lib/core/form/components/widgets/dropdown/dropdown.widget.ts

```diff
@@ -113,4 +113,7 @@ export class DropdownWidgetComponent extends WidgetComponent implements OnInit {
         return this.field.type === 'readonly';
     }
 
+    showRequiredMessage(): boolean {
+        return (this.isInvalidFieldRequired() || this.field.value === 'empty') && this.isTouched();
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Alfresco/alfresco-ng2-components/pullRequest?prid=8980729)
```suggestion
        return (this.isInvalidFieldRequired() || this.field.value === "empty") && this.isTouched();
```